orbstation/xcard/revolution.dm

```diff
@@ -0,0 +1,17 @@
+/// Checks if the given person has the xcard revolution immunity trait. If so, deals damage to them and knocks them unconscious, returning TRUE. Otherwise, returns false.
+/datum/antagonist/rev/proc/check_xcard_and_apply_if_necessary(mob/living/carbon/victim, mob/aggressor)
+
+    if(HAS_TRAIT(victim, TRAIT_XCARD_REV_IMMUNE))
```

Comment:
Probably better as an early return, as in
```
if(!HAS_TRAIT(victim, TRAIT_XCARD_REV_IMMUNE))
     return FALSE

...
```
Then every other line of this method is not indented

Reply:
Edited